sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -25,5 +26,36 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ConnectionString => GetRecordedOptionalVariable("CONNECTION_STRING");
         public string StorageAccountId => GetRecordedOptionalVariable("STORAGE_ID");
         public string StorageAccountConnectionString => GetRecordedOptionalVariable("STORAGE_CONNECTION_STRING");
+
+        public string GetAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                if (AzureAuthorityHosts.AzurePublicCloud.ToString() == LogsQueryAudience.AzurePublicCloud)
+                    return LogsQueryAudience.AzurePublicCloud.ToString();
+                if (AzureAuthorityHosts.AzurePublicCloud.ToString() == MetricsQueryAudience.AzurePublicCloud)
+                    return MetricsQueryAudience.AzurePublicCloud.ToString();
+            }
+
```

Comment:
I think the logic for returning the correct audience is faulty here. In conjunction with the creation of two separate GetAudience functions for metrics and logs, let's simplify the logic in each.
```suggestion
            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
            {
                    return LogsQueryAudience.AzurePublicCloud.ToString();
            }
```